libs/core/likwid/docs/index.rst

```diff
@@ -0,0 +1,18 @@
+..
+    Copyright (c) 2020-2021 The STE||AR-Group
+
+    SPDX-License-Identifier: BSL-1.0
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+.. _modules_likwid:
+
+======
+likwid
+======
+
+TODO: High-level description of the module.
```

Comment:
Would you mind editing this and add a short description of what the module is doing?